portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,23 +14,38 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.datastore.*;
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Date;
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles fetching and posting of comments */
 @WebServlet("/data")
```

Comment:
I'd suggest changing the URL to "/comments".

Also FYI.
Best practice of REST API resource name should be plural.
Further reference can be found here:
https://google.aip.dev/122#collection-identifiers